packages/kbn-shared-ux-components/src/page_template/no_data_page/no_data_card/elastic_agent_card.tsx

```diff
@@ -16,27 +18,28 @@ export const ElasticAgentCard = (props: ElasticAgentCardProps) => {
   const { canAccessFleet } = usePermissions();
   const { addBasePath } = useHttp();
   const { navigateToUrl, currentAppId$ } = useApplication();
+  const currentAppId = useObservable(currentAppId$);
 
-  const createHref = () => {
-    const { href, category } = props;
-    if (href) {
-      return href;
+  const { href: srcHref, category } = props;
```

Comment:
why this rename?

Reply:
So this was the `href` that came on `props`.  I renamed it so I could assign the result of `line 25` to `href`, and use it in the spread on `line 42`.

```tsx
const { something: srcSomething } = props;
const something = `There's that ${srcSomething}`;
return <Component { ...{ something, somethingElse, somethingOther} } />;

// rather than

const { something } = props;
const newSomething = `There's that ${something}`;
return <Component something={newSomething} { ...{ somethingElse, somethingOther} } />;

```